helm-charts/yunikorn/README.md

```diff
@@ -59,33 +59,49 @@ helm install yunikorn yunikorn/yunikorn
+| `admissionController.image.repository`          | Admission controller image repository                | `apache/yunikorn`               |
+| `admissionController.image.tag`                 | Admission controller image tag                       | `admission-latest`              |
+| `admissionController.image.pullPolicy`          | Admission controller image pull policy               | `Always`                        |
+| `admissionController.namespaceBlacklist`        | Comma-separated list of namespace regexes to ignore  | `^kube-system$`                 |
+| `admissionController.resources.requests.cpu`    | Admission controller CPU resource requests           | `100m`                          |
+| `admissionController.resources.requests.memory` | Admission controller memory resource requests        | `500Mi`                         |
+| `admissionController.resources.limits.cpu`      | Admission controller CPU resource limit              | `500m`                          |
+| `admissionController.resources.limits.memory`   | Admission controller memory resource limit           | `500Mi`                         |
+| `admissionController.nodeSelector`              | Admission controller deployment nodeSelector(s)      | `{}`                            |
+| `admissionController.tolerations`               | Admission controller deployment tolerations          | `[]`                            |
+| `admissionController.affinity`                  | Admission controller deployment affinity             | `{}`                            |
+| `service.port`                                  | Port of the scheduler service                        | `9080`                          |
+| `service.portWeb`                               | Port of the web application service                  | `9889`                          |
+| `resources.requests.cpu`                        | CPU resource requests                                | `200m`                          |
+| `resources.requests.memory`                     | Memory resource requests                             | `1Gi`                           |
+| `resources.limits.cpu`                          | CPU resource limit                                   | `4`                             |
+| `resources.limits.memory`                       | Memory resource limit                                | `2Gi`                           |
+| `webResources.requests.cpu`                     | Web app CPU resource requests                        | `100m`                          |
+| `webResources.requests.memory`                  | Web app memory resource requests                     | `100Mi`                         |
+| `webResources.limits.cpu`                       | Web app CPU resource limit                           | `200m`                          |
+| `webResources.limits.memory`                    | Web app memory resource limit                        | `500Mi`                         |
```

Comment:
Same here as for the `webImage` comment we should be consistent and use `web.resources`